python/ray/data/datasource/sql_datasource.py

```diff
@@ -132,6 +135,9 @@ def fallback_read_fn() -> Iterable[Block]:
             is_limit_supported = False
 
         if not is_limit_supported:
+            logger.warning(
+                "Queried Database does not support 'LIMIT', forcing parallelism of 1"
```

Comment:
(Might need to wrap this string to appease lint)
```suggestion
                "Ray Data uses the 'LIMIT' clause to read data in parallel, but the queried database doesn't support 'LIMIT'. Forcing parallelism of 1."
```